buildpipeline/pipeline.json

```diff
@@ -141,8 +140,7 @@
           "Parameters": {
             "PB_AdditionalBuildArguments": "/p:PortableBuild=true /p:SkipTests=true /p:NativeToolSetDir=C:\\tools\\clr",
             "PB_TargetArchitecture": "arm64",
-            "PB_PortableBuild": "true",
-            "PB_DistroRid": "win-arm64"
+            "PB_PortableBuild": "true"
```

Comment:
Distro is only a Linux concept, and that should be cleaned up in the future, but for now, passing the same parameters to `publish` that are passed to `build`, is the right thing to do. 

I think that the better fix, is to make `publish` a part of `build`, but only run it on official builds.  Then we can remove the extra publish step from these build definitions.